### ISSUE TITLE
Hero image in the website does not depict the stock user expericence for Brackets

### ISSUE THREAD

**Issue by abose** [adobe/brackets.io#183]
Just noticed the change in the site, the new hero image of brackets does not seem to be the stock user experience for brackets with a different theme applied and a couple of additional 'extension' buttons. Wouldn't it be better to put in the image of stock brackets as the hero image. It would be confusing for new users to see the program they downloaded does not seem to be the one in the screenshot?

**Comment by ficristo** [comment]
I agree with @abose. If I'm not mistaken there is still the icon of Extract in the right toolbar...
/cc @bethomas
